src/Pages/Product/ProductPage.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import {ProductFilter} from '../../Components/Filter/ProductFilter'
+import {Navbar} from '../../Components/Navigation/Navbar'
+import {ProductList} from '../../Components/ProductList/ProductList'
+
+const ProductPage = () => {
+  return (
+    <div>
```

Comment:
You can use empty brackets <> or Fragment tag, if you don't really require those divs as it will remove unnecessary DOM elements from page